enroll/service.go

```diff
@@ -98,3 +103,58 @@ func (svc service) Enroll() (Profile, error) {
 
 	return *profile, nil
 }
+
+type service struct {
+	Url           string
```

Comment:
`Url` should be `URL`. This applies to all initialisms like `Id` vs` ID` or `Smtp` vs `SMTP` etc.